File: index/src/main/java/com/benxin/service/UserService.java
package com.benxin.service;

import com.benxin.entity.User;
import entity.Result;

public interface UserService {

    /**
     * 注册
     * @param user
     */
    Result regist(User user);

    /**
     * 获取注册验证码
     */
    void getCode(String code);
}

File: index/src/main/webapp/js/service/userService.js
//管理
app.service("userService",function($http){

    this.regist=function(entity){
			return $http.post("../user/regist",entity);
	};
	
	this.sendSms=function (phone) {
		return $http.get("../sms/sendSms?phone="+phone);
    }
});
